examples/lib/defaultHtmlSequencerUi.js

```diff
@@ -47,6 +47,8 @@ function DefaultHtmlSequencerUi(_sequencer, options) {
   
     if(!newStepName) newStepName = arguments[0]
 
+    if(typeof newStepName !== "string")
+    newStepName = $(".item")[0].getAttribute("data-value");
```

Comment:
you can use jQuery `.attr()`. Btw, what is `.item`?

Reply:
Actually this is used because of selectize. Nevertheless, our code is working perfectly.

Reply:
@HarshKhandeparkar is it ok to you?

Reply:
What exactly is `.item` in selectize?

Reply:
Also, you can use `$.attr()` method.

Reply:
Its the class given to a div generated by selectize after  consuming the select tag.

Reply:
I can but that does not make any difference.

Reply:
It shortens the code, makes it look better, makes it consistent

Reply:
```suggestion
    newStepName = $(".item").attr("data-value");
```

As simple as that. Just click that button

Reply:
Let me first try if   its working

Reply:
Btw, How many `.item`s are on the page?

Reply:
Can there be only a single `.item` or more than 1? What about the insert step dropdown. jQuery will search for everything globally.

Reply:
Well just one!!

Reply:
Are you sure? I am talking about the whole DOM.

Reply:
What if there are more than a single selectize selectors?